### ISSUE TITLE
Won't authorize at blogger

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#701]
```
What's the problem?

Won't authorize at blogger





What browser are you using?

firefox 16.0.1



What version of ScribeFire are you running? The newest

```

-----
Original issue reported on code.google.com by `MJAstarita@gmail.com` on 16 Nov 2012 at 2:24

**Comment by GoogleCodeExporter** [comment]
```
I'm having the same problem only with Firefox 18.0.2
```

-----
Original comment by `ho.ol...@gmail.com` on 13 Feb 2013 at 6:54

**Comment by GoogleCodeExporter** [comment]
```
Fixed for blogger blogspot in version 4.3.1
```

-----
Original comment by `scribefi...@gmail.com` on 14 Apr 2014 at 7:01
* Changed state: **Fixed**
